### PR TITLE
Add badge for npm install size

### PR DESCRIPTION
**Pull request by styfle**
This adds a badge to the readme with the install size.

Related: https://github.com/webpack-contrib/webpack-command/pull/9

### CODE REVIEW

**Comment by shellscape** [comment]
@styfle thanks for the PR :beer:

I've requested review by several maintainers. As soon as it's reviewed we'll know whether or not we can move forward.

**Comment by bebraw** [comment]
Yup, that's true. Better modify the template as otherwise the change won't get applied.

**Comment by styfle** [comment]
@shellscape Thanks, hows it look now?

File: README.md [hunk 1]
```diff
@@ -56,3 +57,6 @@ Any time you want to sync your project with the defaults, update this dependency
 
 [cover]: https://codecov.io/gh/webpack-contrib/webpack-defaults/branch/master/graph/badge.svg
 [cover-url]: https://codecov.io/gh/webpack-contrib/webpack-defaults
+
+[size]: https://packagephobia.now.sh/badge?p=webpack-defaults
```
Comment (shellscape) [inline README.md]: please remove these changes from the main README. they won't add any value to webpack-defaults' README

File: templates/README.md [hunk 2]
```diff
@@ -142,4 +143,7 @@ Please take a moment to read our contributing guidelines if you haven't yet done
 [cover-url]: https://codecov.io/gh/webpack-contrib/${package}
 
 [chat]: https://img.shields.io/badge/gitter-webpack%2Fwebpack-brightgreen.svg
-[chat-url]: https://gitter.im/webpack/webpack
\ No newline at end of file
+[chat-url]: https://gitter.im/webpack/webpack
+
+[size]: https://packagephobia.now.sh/badge?p=${package}
```
Comment (shellscape) [inline templates/README.md]: looks good 👍

File: README.md (hunk 1 above)
Comment (shellscape) [reply README.md]: 👍

**Comment by shellscape** [comment]
Thanks for the contribution! You'll see this appear on webpack-contrib repos as they update webpack-defaults (this usually takes some time).

**Comment by styfle** [comment]
Excellent! Thanks!

**Pull request merged**